
＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,467                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14821]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,481                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14835]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,481                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14836]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,482                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14836]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,482                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14837]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,483                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14837]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,483                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14838]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,484                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14838]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,484                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14838]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,484                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14839]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,485                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14839]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,485                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14840]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,486                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14840]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,486                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14841]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:43,601                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[13956]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,487                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14842]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,488                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14842]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,488                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14843]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,260                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14614]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,489                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14844]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,490                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14844]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,490                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14845]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,491                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14845]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,491                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14846]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,492                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14846]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,487                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14841]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,493                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14847]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,493                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14847]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,493                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14848]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,494                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14848]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,494                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14849]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,494                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14849]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,495                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14849]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,495                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14850]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,496                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14850]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,496                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14851]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,497                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14851]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,381                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14735]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,498                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14853]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,499                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14853]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,500                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14854]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,500                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14855]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,501                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14856]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,502                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14856]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,502                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14857]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,503                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14857]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,503                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14858]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,504                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14858]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,504                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14858]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,504                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14859]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,505                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14859]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,505                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14860]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,505                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14860]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,506                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14860]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,506                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14861]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,507                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14861]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,507                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14861]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,507                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14862]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,508                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14862]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,508                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14863]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,508                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14863]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,509                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14863]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,509                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14864]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,510                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14864]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,510                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14865]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,511                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14865]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,511                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14865]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,511                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14866]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,512                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14866]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,512                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14867]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,513                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14867]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,513                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14867]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,513                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14868]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,514                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14868]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,514                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14869]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,515                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14869]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,515                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14869]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,515                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14870]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,516                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14870]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,516                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14871]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,516                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14871]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,517                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14871]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,517                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14872]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,518                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14872]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,518                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14873]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,519                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14873]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,519                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14874]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,519                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14874]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,520                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14874]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,520                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14875]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,521                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14875]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,521                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14875]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,521                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14876]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,522                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14876]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,522                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14877]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,523                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14877]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,523                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14877]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,523                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14878]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,524                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14878]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,524                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14879]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,525                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14879]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,525                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14880]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,526                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14880]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,526                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14880]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,526                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14881]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,527                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14881]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,527                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14882]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,527                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14882]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,528                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14882]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,528                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14883]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,529                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14883]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,529                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14884]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,530                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14884]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26
